src/core/server/elasticsearch/elasticsearch_service.test.ts

```diff
@@ -6,6 +6,9 @@
  * Side Public License, v 1.
  */
 
+// Mocking the module to avoid waiting for a valid ES connection during these unit tests
+jest.mock('./is_valid_connection');
```

Comment:
maybe we should add a test for an incompatible node log message?

Reply:
Sure! I'll do that, although, that's not related to that mocked file :)
Thanks for bringing that up!

Reply:
Done in [`c7395b9` (#107663)](https://github.com/elastic/kibana/pull/107663/commits/c7395b905e6b4bf48ff2ef3a9f0c866b8b232d0c)